docs/examples/managed/GoogleDemo.ipynb

```diff
@@ -211,7 +287,15 @@
    "outputs": [],
    "source": [
     "# Use a previously created corpus.\n",
-    "index = GoogleIndex.from_corpus(corpus_id=\"abc-123\")"
+    "index = GoogleIndex.from_corpus(corpus_id=corpus_id(1))\n",
```

Comment:
This will throw an error, should we use the basic usage one instead here? Like

corpus_id = index.corpus_id
index = GoogleIndex.from_corpus(corpus_id=corpus_id)

Reply:
Ah yes, corpus_id() is currently not deterministic. Fixed.